app/models/micropost.rb

```diff
@@ -6,7 +6,8 @@ class Micropost < ApplicationRecord
     length: {maximum: Settings.validates.max_length_content_micropost}
   validate  :picture_size
   scope :order_desc, ->{order created_at: :desc}
-  scope :feed, ->(id){where(user_id: id)}
+  scope :feed, ->(following_ids, id){where "user_id IN (?) OR user_id = ?",
```

Comment:
`{where "user_id IN (?) OR user_id = ?", following_ids, id}`

-> `do `
`where "user_id IN (?) OR user_id = ?", following_ids, id`
`end`